code/game/objects/structures/bonfire.dm

```diff
@@ -138,6 +138,7 @@
 			if(grill && isitem(burned_object))
 				var/obj/item/grilled_item = burned_object
 				SEND_SIGNAL(grilled_item, COMSIG_ITEM_GRILLED, src, delta_time) //Not a big fan, maybe make this use fire_act() in the future.
+				SEND_SIGNAL(grilled_item, COMSIG_ITEM_DRIED, src, delta_time) //Allows wethide to be dried on bonfire grills
```

Comment:
dried signal is only used by drying racks, adding it to other things to make dryable element work on grilling isn't the way to go. instead...